addons/sms/models/ir_actions_server.py

```diff
@@ -36,6 +36,15 @@ def _compute_name(self):
             )
         super(ServerActions, self - to_update)._compute_name()
 
+    @api.depends('state')
+    def _compute_available_model_ids(self):
+        mail_based = self.filtered(lambda action: action.state == 'sms')
```

Comment:
"mail_thread_based" then? Having "mail_based" and then a check on sms feels strange.